experiments/calculate_fnr.py

```diff
@@ -0,0 +1,33 @@
+from blurring_as_a_service.metrics.custom_metrics_calculator import (
+    CustomMetricsCalculator,
+)
+
+
+def run_custom_metrics_calculator_1mar():
+    yolo_run_path = "../yolov5/runs/val/exp13/"
+    source = f"{yolo_run_path}/labels_tagged"
+    coco_file_with_categories = (
+        "exp_with_tagged_validation_1mar/in:coco-format/validation-tagged.json"
+    )
+    markdown_output_path = (
+        f"{yolo_run_path}/custom_metrics_result_imgsz_640_with_tba.md"
+    )
+
+    metrics_calculator = CustomMetricsCalculator(source, coco_file_with_categories)
+    metrics_calculator.calculate_and_store_metrics(markdown_output_path)
+    # metrics_calculator._plot_area_distribution(path=".", plot_name="GT_bboxes_2")
```

Comment:
To avoid commenting and uncommenting you could set up a parameter as input called plot.
And then with an if check if needs to plot or not ;)